doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -381,35 +367,41 @@ Modem libraries
 
   * :ref:`lib_location` library:
 
-    * Added support for P-GPS data retrieval from an external source, implemented separately by the application.
-      Enabled by setting the :kconfig:option:`CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL` option.
-      The library triggers a :c:enum:`LOCATION_EVT_GNSS_PREDICTION_REQUEST` event when assistance is needed.
-    * The :c:member:`request` member of the :c:struct:`location_event_data` structure was renamed to :c:member:`agps_request`.
-    * Current system time is attached to the ``location_datetime`` parameter of the location request response with Wi-Fi and cellular methods.
-      The timestamp comes from the moment of scanning or neighbor measurements.
-    * Removed dependency on the :ref:`lib_modem_jwt` library.
-      The :ref:`lib_location` library now selects :kconfig:option:`CONFIG_NRF_CLOUD_REST_AUTOGEN_JWT` when using :kconfig:option:`CONFIG_NRF_CLOUD_REST`.
-    * Added a new feature, obstructed satellite visibility detection for GNSS.
-      When this feature is enabled, the library tries to detect occurrences where getting a GNSS fix is unlikely or would consume a lot of energy.
-      When such an occurrence is detected, GNSS is stopped without waiting for a fix or a timeout.
+    * Added:
+
+      * Support for P-GPS data retrieval from an external source, implemented separately by the application.
+        Enabled by setting the :kconfig:option:`CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL` option.
+        The library triggers a :c:enum:`LOCATION_EVT_GNSS_PREDICTION_REQUEST` event when assistance is needed.
+      * A new feature, obstructed satellite visibility detection for GNSS.
```

Comment:
```suggestion
      * Obstructed satellite visibility detection feature for GNSS.
```
Or "A new feature: obstructed...". But I think the suggestion might be better, as it is in the Added section, so it is somewhat obvious we're talking about a new feature.

Reply:
Fixed